### PR TITLE
8310330: HttpClient: debugging interestOps/readyOps could cause exceptions and smaller cleanup

### PR DESCRIPTION
**Pull request by dfuch**
Please find here a change that makes sure that debugging interestOps/readyOps won't cause exceptions to be raised.
The change also contains smaller cleanup: extracting ALPN contants to their own class, removing/adding newlines at end of files, avoiding long lines in some occasions, removing some unused imports, etc...

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8310330](https://bugs.openjdk.org/browse/JDK-8310330): HttpClient: debugging interestOps/readyOps could cause exceptions and smaller cleanup (**Bug** - P4)

### Reviewers
 * [Daniel Jeliński](https://openjdk.org/census#djelinski) (@djelinski - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/14546/head:pull/14546` \
`$ git checkout pull/14546`

Update a local copy of the PR: \
`$ git checkout pull/14546` \
`$ git pull https://git.openjdk.org/jdk.git pull/14546/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 14546`

View PR using the GUI difftool: \
`$ git pr show -t 14546`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/14546.diff">https://git.openjdk.org/jdk/pull/14546.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/14546#issuecomment-1597520003)

### CODE REVIEW

File: src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java [hunk 1]
```diff
@@ -276,7 +279,7 @@ public void log(Level level, ResourceBundle unused, String msg,
                 && logLevel.getSeverity() <= severity
                 && logger.isLoggable(level)) {
             logger.log(level, unused,
-                    getFormat(new StringBuilder(), msg, null).toString(),
+                    format(new StringBuilder(), msg, null).toString(),
```
Comment (djelinski) [inline src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java:282]: `getFormat` is no longer used, can you remove it?

**Review by djelinski — APPROVED** [review]
LGTM. 2 minor issues reported in line. Please update copyright before integrating.

File: src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java (hunk 1 above)
Comment (dfuch) [reply src/java.net.http/share/classes/jdk/internal/net/http/common/DebugLogger.java:282]: Oh - that's an oversight indeed. Thanks for spotting it. Done.

**Pull request closed without merging**